### PR TITLE
Added option to have circular hitboxes

### PR DESCRIPTION
**Pull request by milosmilosevic13**

### CODE REVIEW

**Comment by milosmilosevic13** [comment]
I'm not sure if the GetCollisionSides and GetCollisionCenter are affected by this change, so I left them unchanged for now.

File: source/dagger/gameplay/common/simple_collisions.cpp [hunk 1]
```diff
@@ -3,6 +3,9 @@
 #include "core/engine.h"
 #include "core/game/transforms.h"
 
+#include <iostream>
```
Comment (DPrivalov) [inline source/dagger/gameplay/common/simple_collisions.cpp:6]: do u need this?

**Review by DPrivalov — APPROVED** [review]
try to split into more smaller pull requests next time

File: source/dagger/gameplay/common/simple_collisions.cpp (hunk 1 above)
Comment (milosmilosevic13) [reply source/dagger/gameplay/common/simple_collisions.cpp:6]: No, it was used for some debugging but wasn't removed afterwards, my bad.

**Review by nkristina — APPROVED** [review]

**Review by mmacura9 — APPROVED** [review]

**Pull request merged**